test/system/700-play.bats

```diff
@@ -411,6 +411,23 @@ _EOF
     run_podman rmi -f userimage:latest
 }
 
+@test "podman kube play --replace external storage" {
+    TESTDIR=$PODMAN_TMPDIR/testdir
+    mkdir -p $TESTDIR
+    echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
+    run_podman play kube $PODMAN_TMPDIR/test.yaml
+    # Force removal of container
+    run_podman rm --force -t0 test_pod-test
+    # Create container external using buildah with same name
+    buildah from --name test_pod-test $IMAGE
```

Comment:
This probably needs a `run buildah` and status checks.

Reply:
Not really. `buildah` here is much better than `run buildah`.

Reason: `run` is tricky. It hides exit status and output. In this case here, if plain `buildah` fails, BATS will immediately fail and will show all stdout/stderr. If you `run buildah`, though, you then need to add cumbersome `if $status blah` checks, and then will miss stdout/stderr if you check in the wrong order.